docs/mixin/dashboards/promscale.json

```diff
@@ -2887,7 +2887,7 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "exemplar": true,
-              "expr": "promscale_cache_query_hits_total{type=\"metric\"} / promscale_cache_queries_total{type=\"metric\"}",
+              "expr": "rate(promscale_cache_query_hits_total{type=\"metric\"}[$__rate_interval]) / rate(promscale_cache_queries_total{type=\"metric\"}[$__rate_interval])",
```

Comment:
Can we use irate for graphs instead of rate?

https://www.robustperception.io/irate-graphs-are-better-graphs/

Reply:
We could, but irate is very sensitive to instant spikes. I am not confident that we need such instant spikes for cache hits.

Another reason being, see the graphs, `irate` (top) vs `rate` (bottom) at the drop just after 12:55 (that's where i added more series to replicate a cache miss)

<img width="1294" alt="Screenshot 2022-10-06 at 1 01 34 PM" src="https://user-images.githubusercontent.com/33792202/194244954-0ca880e7-6c4c-4c16-85aa-98eac08192c3.png">

<img width="1294" alt="Screenshot 2022-10-06 at 1 01 25 PM" src="https://user-images.githubusercontent.com/33792202/194245365-f7ae8fc0-2e6c-4372-9966-5c4306697ef7.png">

As evident, `irate` is so sensitive, that it just cared about the 1 step evaluation of range query, and just showed a drop denoted by `1 sample`. On the contrary, the hits went down was justified by `rate`, since the averaging logic reflects the drop in _samples evaluated (for `rate`) after the actual drop_.

Given this behaviour, I think we should continue with `rate`.